plot.py

```diff
@@ -0,0 +1,37 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# df = pd.read_csv("test_for_plot.txt", sep="\t")
+# df['mean_rain'] = [0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 11.0, 20.0, 17.0,
+#                    6.0, 0.5, 0.2, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0,
+#                    0.0, 0.0, 0.0, 0.2, 0.1, 0.2]
+
+
+def create_plot(df, parameters):
+    # create plot axes limits
+    max_rain = 10
+    rain_max = df["mean_rain"].max()*1.1
+    if max_rain < rain_max:
+        max_rain = rain_max
+
+    max_temp = df["mean_temp"].max()*1.1
+    min_temp = df["mean_temp"].min() - df["mean_temp"].max()*0.1
```

Comment:
Why/what are `10`, `1.1`, or `0.1`? Below I'm also seeing `12`, `20`, `16`, and `300`. Again, these are "magic numbers". Please handle that in accordance with clean code guides.

Reply:
@KrzysiekJodlowski 
Here I would need a little bit of guidance from you since I will not start to read a book right now. The first set of numbers that you mention is arbitrarily chosen to make the precipitation plot more intuitive. When rain is small or non then scale will keep 10 mm as max and not stretch bars up. Also, max scale precipitation will be always 10% higher than max measurement for clarity so the top of the bars will not touch the top of the plot.
Another set of numbers is responsible for font size or plot size.
What should I do? Explain it in the comments?

Reply:
@deerhunter4 let me make that easier for you 😉 You can start by looking at this webpage: https://refactoring.guru/pl/replace-magic-number-with-symbolic-constant

Reply:
Done. I created constant values only for the first three "magic numbers', plus I commented it very well. 
Other numbers for me are self-explanatory like 'fontsize' and others needed just a few words of explanation as a commented text near to them.